### PR TITLE
#213 Implemented DSE Type Codecs

### PR DESCRIPTION
**Pull request by Jeremya**
**What this PR does**: Add DSE Types support to CDM like [Geotypes](https://docs.datastax.com/en/developer/java-driver/4.13/manual/core/dse/geotypes/) and DateRange type.

**Which issue(s) this PR fixes**: #213 

**Checklist:**
- [ ] Automated Tests added/updated
- [ ] Documentation added/updated
- [x] CLA Signed:  [DataStax CLA](https://cla.datastax.com/)

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -143,6 +143,11 @@
       <artifactId>log4j-to-slf4j</artifactId>
       <version>${log4j.version}</version>
     </dependency>
+    <dependency>
```
Comment (msmygit) [inline pom.xml:146]: try scoping it just to tests?

**Review by faizalrub-datastax — APPROVED** [review]
Please ensure that both the migration and validation jobs are tested. Also, include any unit and integration tests.

File: pom.xml (hunk 1 above)
Comment (Jeremya) [reply pom.xml:146]: Good idea, on it.

**Review by msmygit — APPROVED** [review]

**Comment by msmygit** [comment]
@all-contributors please add @Jeremya for code

**Pull request merged**